app/components/post_list/post/avatar/index.tsx

```diff
@@ -11,7 +11,7 @@ import {of as of$} from 'rxjs';
 import {switchMap} from 'rxjs/operators';
 
 import CompassIcon from '@components/compass_icon';
-import ProfilePicture from '@components/profile_picture';
+import UserProfilePicture from '@components/profile_picture/image';
```

Comment:
this is throwing me off.. should it be something like
```suggestion
import ProfileImage from '@components/profile_picture/image';
```